test/integration/signup_express_test.rb

```diff
@@ -18,10 +27,6 @@ class SignupExpressTest < ActionDispatch::IntegrationTest
   end
 
   test 'signup express with account plan that do not requires approval' do
-    @account_plan = FactoryGirl.create(:account_plan, approval_required: false)
-    @provider = @account_plan.provider
-    host! @provider.admin_domain
-
     post '/admin/api/signup.xml', provider_key: @provider.api_key, org_name: 'company', username: 'quentin', email: 'quentin@example.com', password: '12345678', account_plan_id: @account_plan.id
```

Comment:
No longer testing with `approval_required: false`?

Reply:
sure. the test is falling. will add that back.